src/gql/generated/types.ts

```diff
@@ -3912,10 +3912,7 @@ export type GetUserConfigQuery = {
 export type GetUserPermissionsQueryVariables = Exact<{ [key: string]: never }>;
 
 export type GetUserPermissionsQuery = {
-  user: {
-    userId: string;
```

Comment:
I think this `userId` should stay to help with Apollo caching (maybe @khelif96 can confirm)

Reply:
Yes userId should stay for the user object. My earlier comment was only in reference to the permissions object.

Reply:
restored